examples/multiclass_classification/loss.weights

```diff
@@ -0,0 +1,5 @@
+0.00000	0.67822	0.66155	0.25288	0.24816
```

Comment:
Is everything OK with this file? Seems that the first and second columns are merged.

Reply:
Sorry, I'm not sure what you mean about the columns being merged? When I look at the file in my editor it shows the whitespace characters are all tabs.

Reply:
Ah, seems that it's a GitHub issue then - the first tab (which separates the first and second columns) is not visible at all:

![image](https://user-images.githubusercontent.com/25141164/69811518-462e6600-11ff-11ea-9f0b-097b3f5442b3.png)